mu-plugins/central-hub/src/config-store/api.php

```diff
@@ -126,5 +126,11 @@ function getAllKeysStartingWith( $starts_with ) {
 		}
 	}
 
+	if ( empty( $filtered_keys ) ) {
+		throw new \InvalidArgumentException(
```

Comment:
@rgadon107 I don't think this is the right response.  I think the severity of what to do when a configuration doesn't exit belongs to the code that calls this function.  For example, maybe there aren't any metaboxes or custom post types to be registered.  In that case, it's up to the Metabox or CPT code to decide if shutting down the website with a fatal error is the right response.  Don't you agree?

Reply:
@hellofromtonya I see your point, and it makes sense. Let me go back, retrace the code, and think about the context in which the API function `getAllKeysStartingWith` is used.